packages/botonic-react/src/webchat/components/persistent-menu.jsx

```diff
@@ -35,7 +35,9 @@ export const OpenedPersistentMenu = ({ onClick, options }) => {
           )
         )
       })}
-      <Button onClick={onClick}>{closeLabel}</Button>
+      <Button onClick={onClick} bottom={borderRadius}>
```

Comment:
Could you explain why a radius is applied to a position (`bottom`) property?

Reply:
Inside the Button the bottom radius are defined as:
` border-bottom-right-radius: ${props => props.bottom || '0px'};`
` border-bottom-left-radius: ${props => props.bottom || '0px'};`
So I have used the same name, is it more clear if I use `bottomRadius` instead?

Reply:
I think so, yes. I got confused because there's the `bottom` css property. I thought you were setting that position on the button.